docs/src/getting-started.md

```diff
@@ -0,0 +1,133 @@
+# Getting Started
+
+## For Users
+
+### Installation
+
+First, download and install Julia by following the instructions at [https://julialang.org/downloads/](https://julialang.org/downloads/).
+Then, you can install the ClimaLand package by doing:
+
+```julia
+julia> ] # Enter Package REPL mode
+Pkg> add ClimaLand # Install ClimaLand
+Pkg> # Go back to Julia REPL mode
+Julia> using ClimaLand
+```
+
+### Parameterisation
+
+Let's start with an basic example: compute canopy gross photosynthesis (GPP).
+
+```@repl
+using ClimaLand
+@doc ClimaLand.Canopy.compute_GPP
+```
+
+As you can see, our parameterisation for GPP is located in the `Canopy` Module, and requires four FT.
```

Comment:
I think users should not be worried about `FT` at this point. Let's introduce this complication later.